packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js

```diff
@@ -149,8 +148,7 @@ describe('ReactDOMFiberAsync', () => {
   describe('concurrent mode', () => {
     beforeEach(() => {
       jest.resetModules();
-      ReactFeatureFlags = require('shared/ReactFeatureFlags');
-      ReactFeatureFlags.debugRenderPhaseSideEffectsForStrictMode = false;
```

Comment:
Can remove  `internal` suffix now

Reply:
Yea I'll do a follow up to rename all these if I land this. I think the remaining ones are just using the `let ops = []` pattern and we can port those.